csi/controller.go

```diff
@@ -49,7 +51,8 @@ const (
 	volumeCapabilityMessageNotMultinodeVolume = "Volume is not a multinode volume"
 	volumeCapabilityMessageReadOnlyVolume     = "Volume is read only"
 	volumeCapabilityMessageNotReadOnlyVolume  = "Volume is not read only"
-	defaultCSIVolumeSize                      = uint64(1024 * 1024 * 1024)
+	defaultCSIVolumeSize                      = uint64(GiB * 1)
+	GiB                                       = 1024 * 1024 * 1024
```

Comment:
can we use https://github.com/libopenstorage/openstorage/tree/master/pkg/units instead?

Reply:
Updated